build.gradle

```diff
@@ -29,42 +17,20 @@ plugins {
 repositories {
     mavenLocal()
     mavenCentral()
-//    maven {
-//        url = 'https://jitpack.io'
```

Comment:
FYI-- I don't know what this was used for in the Maven build so I just brought it over-- we should confirm that it's not needed anymore. @blentz @neutronc ?

Reply:
Dont know may come from some automatism from eclipse

Reply:
`jitpack` is a package repository that allows including projects that are hosted on github as a dependency. As far as I know we don't use a dependency that is only hosted on github. Therefore I think we can remove this.